docs/get-started/quickstarts/detect-people.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "Detect People with a Webcam"
+linkTitle: "Detect people with a webcam"
+weight: 50
+type: "docs"
+tags: ["vision", "data", "services", "quickstart", "ml", "camera", "webcam"]
+no_list: true
+description: "Use Viam's machine learning capabilities to deploy a person detector ML model to your machine."
+images: ["/platform/ml.svg"]
+imageAlt: "Machine Learning"
+---
+
+Follow this guide to detect people with a webcam on the Viam platform, using a [pre trained ML model from the registry](https://app.viam.com/ml-model/ml-models-scuttle/people).
+
+## Requirements
+
+- A computer: a development machine like your laptop and optionally a single-board computer like the Raspberry Pi
+- A webcam: this could be the webcam on your laptop or any other webcam you can connect to your computer
```

Comment:
Opt: Non-webcam cameras could work here as well, but the suggestion of a webcam to start is definitely the way to go. Perhaps something like:

```suggestion
- A camera: the webcam on your development machine or a camera device connected to your single-board computer
```

However, if we want to orient this whole quickstart towards webcams specifically, I'm ok with that, disregard this comment!

Reply:
I was thinking webcam specifically bc it was was labeled as so [on ticket ](https://viam.atlassian.net/jira/software/c/projects/DOCS/boards/31?quickFilter=40&selectedIssue=DOCS-2481)

Reply:
Right on, works for me!